src/components/Vuetable.vue

```diff
@@ -650,6 +655,16 @@ export default {
 
       return opacity
     },
+    applyLink (value, item, field) {
+      if (field.link && field.link.length) {
+          let expr = /\{(.+)\}/
+          let linkField = expr.exec(field.link)[1]
```

Comment:
Not sure what this exactly does but it looks very error prone to get the 1st item from that array. What if it doesn't exist or there is no match ? Maybe verify first that the array result has a length > 1